server/tests/controllers/quotes_controller.test.ts

```diff
@@ -61,4 +61,40 @@ describe("QuotesController", () => {
         });
     });
   });
+
+  describe("group", async () => {
+    let fakeQuotes = [
+      [
+        { quote: "Fake", author: "Fake Author" },
+        { quote: "Fake 2", author: "Fake Author 2" }
+      ],
+      [
+        { quote: "Fake", author: "Fake Author" },
+        { quote: "Fake 2", author: "Fake Author 2" }
+      ],
+      [
+        { quote: "Fake", author: "Fake Author" },
+        { quote: "Fake 2", author: "Fake Author 2" }
+      ],
+      [
+        { quote: "Fake", author: "Fake Author" },
+        { quote: "Fake 2", author: "Fake Author 2" }
+      ]
+    ];
+
+    beforeEach(() => {
+      // Stub the output of Quotes to provide a specific quote grouping
```

Comment:
Very awesome! This is really good seperation of concerns. Here we aren't testing the output of `Quotes.getGroup` but rather that whatever it returns if provided in the response.

🚀 👍